pkg/runtime/runtime.go

```diff
@@ -747,6 +761,22 @@ func (a *DaprRuntime) initPubSub() error {
 	if a.pubSub != nil && a.appChannel != nil {
 		topics := a.getSubscribedTopicsFromApp()
 		for _, t := range topics {
+			allowed := false
+
+			if len(allowedSubscriptions) == 0 {
+				allowed = true
+			} else {
+				for _, s := range allowedSubscriptions {
+					if s == t {
+						allowed = true
+						break
+					}
+				}
+			}
+			if !allowed {
+				continue
```

Comment:
@yaron2 should this not be shown as an error message at the logs?
e.g.
```
fmt.Errorf("subscription %s is not allowed for app id %s", s, a.runtimeConfig.ID)
```

Reply:
As a warning, yes.